activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -89,8 +89,8 @@ def after_teardown
       #     User.create.created_at # => Sun, 10 Nov 2013 15:34:49 EST -05:00
       #   end
       #   Time.current # => Sat, 09 Nov 2013 15:34:49 EST -05:00
-      def travel(duration, &block)
-        travel_to Time.now + duration, &block
+      def travel(duration, with_usec: false, &block)
+        travel_to Time.now + duration, with_usec: with_usec, &block
```

Comment:
Hey @serprex thanks for the PR. This needs a few things before we can merge.

* Can you add tests for these two methods that test the `with_usec` argument?
* This needs a changelog entry
* The docs above `travel` and `freeze_time` need to be updated.

Reply:
Done. #44088 didn't add changelog entry, so added changelog entry which addresses both that PR & this follow up

Reply:
Awesome, thank you! Can you squash your commits and then I'll merge?

Reply:
Done